dss/util/auth/fusillade.py

```diff
@@ -23,28 +32,17 @@ def security_flow(self, *args, **kwargs):
         """
         This method maps out security flow for Auth with Fusillade.
 
-        The current implementation of Fusillade (2.0+) uses three pieces of information
+        The current implementation of Fusillade (2.x) uses three pieces of information
         to evaluate authorization: principals, actions, and resources.
 
         However, we have overridden this with a simpler authentication-based
         authorization layer that just checks for membership in a group.
         """
-        # Get token
-        kwargs['security_token'] = request.token_info
-
-        # Set security_groups using first argument, if kwarg not present
         if kwargs.get('security_groups') is None:
-            kwargs['security_groups'] = args[0]
-
-        # Verify JWT was populated correctly
-        self.assert_required_parameters(kwargs, ['security_groups', 'security_token'])
-        groups = kwargs.get('security_groups')
-        token = kwargs.get('security_token')
-
-        # Import when this method is called, not when it is defined, to avoid circular imports
-        from ..security import assert_authorized_group
-        assert_authorized_group(groups, token)
-
+            groups = args[0]
```

Comment:
It seems that we are reusing this block of code from above, just a nit, perhaps there should be a helper method that can be called to perform this operation.

Reply:
Moved this to a mixin class in #126